frontend/public/components/environment.jsx

```diff
@@ -298,6 +309,7 @@ export const EnvironmentPage = connect(stateToProps)(
         success: null,
         modified,
       });
+      onChange(currentEnv.dispatchNewEnvironmentVariables());
```

Comment:
Do we need to check if `onChange` is defined?

Reply:
@spadgett we don't because i defaulted onChange to a noop function from lodash

Reply:
OK, I'd still suggest

```suggestion
      _.isFunction(onChange) && onChange(currentEnv.dispatchNewEnvironmentVariables());
```

Then remove the defaulting. That way, we don't run the `dispatchNewEnvironmentVariables` logic if there is no `onChange` prop